### PR TITLE
Proper check for destruction of global interface object

### PR DESCRIPTION
**Pull request by ellert**
"if (gR) { ... }" will not work as a check whether the global interface object is freed. gR - the pointer to the TRInterface object - is assigned by new. Only one instance of this class can be created. There is no explicit delete of this global object, but it is destroyed when the program exits. But this destruction of remaining object when the program ends does not reset the gR pointer to NULL, so using "if (gR) { ... }" as a test to check whether the object is destroyed or not will not work. I think this PR implements the intended behaviour.

### CODE REVIEW

**Comment by Axel-Naumann** [comment]
Thanks, Mattias!

@phsft-bot build!

File: bindings/r/src/TRInterface.cxx [hunk 1]
```diff
@@ -53,6 +53,7 @@ TRInterface::TRInterface(const Int_t argc, const Char_t *argv[], const Bool_t lo
 
 TRInterface::~TRInterface()
 {
+   statusEventLoop = kFALSE;
```
Comment (pcanal) [inline bindings/r/src/TRInterface.cxx]: Should'nt this also contains
```
if (gR == this) gR = nullptr;
```
?

**Comment by peremato** [comment]
@phsft-bot build just on slc6/gcc62

**Comment by Axel-Naumann** [comment]
@lmoneta could you rebase instead of merging?

**Pull request merged**